dlp/snippets/inspect/inspect_file.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package inspect
+
+// [START dlp_inspect_file]
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+func inspectFile(projectID, filepath, fileType string) error {
+	// projectID := "my-project-id";
+	// filePath := "path/to/image.png";
+	// fileType := "IMAGE"
+
+	// Initialize client
+	client, err := dlp.NewClient(context.Background())
+	if err != nil {
+		return err
+	}
+	defer client.Close() // Closing the client safely stops any background threads or connections.
+
+	// Prepare the request
+	// Set the item for the request.
+	data, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return err
+	}
+	var itemType dlppb.ByteContentItem_BytesType
```

Comment:
TODO(tbpg): String value for this proto?

Reply:
(Untested.)

```suggestion
	itemType := dlppb.ByteContentItem_BytesType_value[fileType].(ByteContentItem_BytesType)
```

Reply:
`itemType := dlppb.ByteContentItem_BytesType_value[fileType].(dlppb.ByteContentItem_BytesType)` leads to an invalid type assertion because "non-interface type int32 on left".

It looks like I could the following:
`itemType := interface{}(dlppb.ByteContentItem_BytesType_value[fileType]).(dlppb.ByteContentItem_BytesType)`. Would that be preferred over a switch?